Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -211,6 +212,7 @@ class LibraryViewController: UIViewController, Themeable {
                               accessibilityLabel: accessibilityLabel,
                               accessibilityIdentifier: accessibilityId)
             self.showPanel(navigationController)
+            navigationHandler?.start(panelType: viewModel.selectedPanel ?? .bookmarks, navigationController: navigationController)
```

Comment:
This feels like it should be behind the library coordinator flag? So in the case the flag is on, we show panel with the `navigationHandler`, and in the else case we handle it with the current method. So we protect our changes behind the flag in production code + much easier to know what we can clean up when the time comes

Reply:
ok perfect, then i'm going to protect it